charts/sentence-collector/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
Thanks for checking in the lock files :)

Reply:
I'm doing it actually after a conversation beween you and Adam regarding Python lockfiles :)